### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

File: online-services-ui/src/app/app.component.spec.ts [hunk 1]
```diff
@@ -16,10 +17,11 @@ describe('AppComponent', () => {
     TestBed.configureTestingModule({
       imports: [
         RouterTestingModule,
+        TranslateModule.forRoot(),
         BrowserAnimationsModule],
 
       declarations: [AppComponent],
-      schemas: [NO_ERRORS_SCHEMA]
+      schemas: [NO_ERRORS_SCHEMA],
```
Comment (99999989) [inline online-services-ui/src/app/app.component.spec.ts]: Unnecessary comma

File: online-services-ui/src/app/app.component.ts [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 
 @Component({
   selector: 'os-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
+  constructor(translate: TranslateService) {
+    // this language will be used as a fallback when a translation isn't found in the current language
+    translate.setDefaultLang('en');
```
Comment (99999989) [inline online-services-ui/src/app/app.component.ts]: Please put this logic into ngOnInit() method.

File: online-services-ui/src/app/core/version/version.component.html [hunk 3]
```diff
@@ -1 +1 @@
-ui: {{frontendVersion}}, backend: {{backendVersion}}
+<div>ui: {{frontendVersion}}, backend: {{backendVersion}}</div>
```
Comment (99999989) [inline online-services-ui/src/app/core/version/version.component.html]: If you add an element that does not need to be rendered on the page, use `<ng-container>` (if possible).

File: online-services-ui/src/app/online-forms/forms/avp-form/employments/employments.component.html [hunk 4]
```diff
@@ -7,6 +7,8 @@
       <os-radio-button [control]="form.get('worked')">
       </os-radio-button>
     </os-panel>
+    <!--<p>{{'HELLO'|translate}}</p>-->
```
Comment (99999989) [inline online-services-ui/src/app/online-forms/forms/avp-form/employments/employments.component.html]: Please remove this comment

File: online-services-ui/src/app/app.component.ts (hunk 2 above)
Comment (ganqqwerty) [reply online-services-ui/src/app/app.component.ts]: not in this case, that's from TranslateService manual

File: online-services-ui/src/app/core/version/version.component.html (hunk 3 above)
Comment (ganqqwerty) [reply online-services-ui/src/app/core/version/version.component.html]: done

File: online-services-ui/src/app/online-forms/forms/avp-form/employments/employments.component.html (hunk 4 above)
Comment (ganqqwerty) [reply online-services-ui/src/app/online-forms/forms/avp-form/employments/employments.component.html]: done

File: online-services-ui/src/app/app.component.spec.ts (hunk 1 above)
Comment (ganqqwerty) [reply online-services-ui/src/app/app.component.spec.ts]: done

**Pull request merged**